permissions.cfg

```diff
@@ -43,3 +43,7 @@ access_level sv_map 1
 access_level sv_timelimit 1
 access_level swap_teams 1
 access_level swap_teams_random 1
+access_level sv_force_ready_all 1
+access_level pause_game 1
+access_level sv_force_ready 1
```

Comment:
the rcon command is called ``force_ready`` only config variables have the sv_ prefix

Reply:
oop